hackathon/urls.py

```diff
@@ -1,9 +1,11 @@
 from django.urls import path
-
+from . import views
```

Comment:
Could you be more explicit in importing your views?

Reply:
In the new django lessons Chris is importing only the views and using views.judging in the path. I thought it was nicer, but prefer importing the function as you suggested 😉